buildSrc/src/main/kotlin/io/getstream/chat/android/Dependencies.kt

```diff
@@ -65,7 +65,7 @@ object Versions {
     internal const val MOCKITO_KOTLIN = "2.2.0"
     internal const val MOCKITO = "3.11.2"
     // Be careful with updating Moshi to 1.13.0 as it's shipped as MR-jar which causes problems with Jetifier. Check [this](https://github.com/square/moshi/issues/1424) issue for more details
-    internal const val MOSHI = "1.12.0"
+    internal const val MOSHI = "1.13.0"
```

Comment:
Had to bump up Moshi because Compose 1.1.0-rc03 requires Kotlin 1.6.10 and  Moshi 1.12.0 fails to parse metadata if we are using Kotlin 1.6.10